playgrounds/angular/src/index.html

```diff
@@ -7,7 +7,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
 </head>
-<body>
+<body class="dx-material-blue-light">
```

Comment:
Question: Do we really need this class, because react works without it?

Reply:
Discussed.

Maybe we should leave only the scoped version of the CSS styles.
But it needs to be discussed in the future, so I fixed this and made it consistent with current React examples.